virtcontainers/qemu.go

```diff
@@ -711,6 +712,10 @@ func (q *qemu) addDeviceToBridge(ID string) (string, Bridge, error) {
 		}
 	}
 
+	if err == nil {
+		err = errors.New("failed to get available address from bridges")
+	}
```

Comment:
This looks sane to me, but please simplify this with:
```go
return fmt.Errorf("failed to find any available address from bridges")
```

Reply:
Yeah I get it, then one clean way to write it would be:
```go
func (q *qemu) addDeviceToBridge(ID string) (addr string, br Bridge, err error) {
	if len(q.state.Bridges) == 0 {
		return
	}

	// looking for an empty address in the bridges
	for _, b := range q.state.Bridges {
		addr, err := b.addDevice(ID)
		if err == nil {
			return fmt.Sprintf("%02x", addr), b, nil
		}
	}

	return addr, br, fmt.Errorf("failed to find any available address from bridges: %v", err)
}
```